index.html

```diff
@@ -221,13 +268,16 @@ <h2 class="contact-heading">
           <textarea name="contact" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
           <div class="form-button"><input type="submit" value="Get In Touch" class="contact-button"></div>
         </form>
+       
+        </div>
         <hr/>
+        <footer>
         <div class="contact-icon">
           <img src="./Pictures/SocialMedia2.png" alt="Social media">
         </div>
         <div class="shape">
           <img src="./Pictures/Shape.png" alt="shape">
         </div>
-    </div>
+    </footer>
 </body>

```

Comment:
- [ ] Kindly consider fixing the design of the contact to match the Figma template.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/200133018-7147ada7-2091-4200-b1e5-4069a86deade.png)  | ![image](https://user-images.githubusercontent.com/53402600/200133004-d15c16a0-f647-415d-895d-69831e75a11a.png) |